src/kiwixapp.cpp

```diff
@@ -208,11 +208,21 @@ void KiwixApp::restoreTabs()
       {
         try
         {
+          if (zimUrl == "SettingsTab")
+          {
+            getTabWidget()->openOrSwitchToSettingsTab();
+            continue;
+          }
```

Comment:
I think this should be outside the `try` block

Reply:
Try block is not needed since we are no longer checking if a Zim file exist.